### DISCUSSION TITLE
投げ銭機能がほしい

### DISCUSSION THREAD

**Discussion by munierujp** [increments/qiita-discussions#64] [要望・アイデア・不具合]
YouTubeやTwitterなどのCGM的なサービスにおいて、いわゆる投げ銭機能の存在は一般的になりつつあります。
エンジニア界隈でもGitHub Sponsorsがあったり、Qiitaの競合サービスでは既に投げ銭機能が存在しているものもあります。

投げ銭機能の存在は、記事を書く側としてはコントリビューションにくわえてモチベーションの一要素となりえます。
また、記事を読む側としても特に役に立った記事にはLGTMだけでなく投げ銭をして感謝を示せるといいと思います。
一方で、「投げ銭目当ての投稿が増えるのではないか」といった懸念もあるでしょう。

みなさんはどう思いますか？

**Comment by kskwtnk** [comment]
@munierujp 

ご意見＆投げかけありがとうございます！
少し長文になりますが運営としての考えを回答いたします 🙏 

まず結論としては、現時点では投げ銭を実装するための具体的なロードマップなどは引いていません
ただし「投げ銭機能を実装つもりは無い」わけではなく、もう少し詳細な背景をご説明します

前提として、投稿者のみなさんに対して還元したい気持ちは強く、そのためにはどういった方法が良いのかは長らく議論しています
もちろん投げ銭の話も出るのですが、投げ銭は不可逆的かつ、今のコミュニティの雰囲気を破壊してしまうリスクもあるのではないか……そう考えています

運営としては即物的な投げ銭よりも「Qiitaで活動し続けることがエンジニアとしての資産になっていく」という世界観を作りたいと思っており、例えば

- 本の出版、寄稿などのその人が記事を書くこと自体にインセンティブが生まれる世界
- イベントなどへの登壇

といったものが挙がっています

そのため今は「投げ銭をやるかどうか」というよりも「記事投稿者にどういう価値を還元できるか」という軸で日々議論をしている最中です

と、ここまで一方的に運営の考えている内容を記載していますが、皆さんの意見は是非いただきたいです！
よろしくお願いします！

**Comment by yimajo** [comment]
@xrxoxcxox 

もう何年もずっと気になっていたことなので運営サイドのご意見が聞けてとてもありがたいです。

> 今のコミュニティの雰囲気を破壊してしまうリスクもあるのではないか……

コミュニティの雰囲気を破壊というのは私にはあまり良くわかってはないのですが、あくまでコミュニティの雰囲気の良し悪しという意味で、noteとかzennの雰囲気が悪いとも思えないですし、むしろ良い意味で活発じゃないかと感じます。体験済みかもしれませんが、Qiita運営の方々にはzennとかnoteで有料で記事書いて体験してほしいなとも思います。

**Reply by kskwtnk** [reply]
@yimajo 

ご意見ありがとうございます！

破壊、という言い方があまりよろしくなかったです 🙏 

雰囲気を「悪くする」といった意味ではなく、今のQiitaにある「他者にとっても役立つ記事を提供する」という空気を大事にしたいといった意味合いでした
コミュニティ規模が大きくて、かつ投げ銭のような機能があるとお金だけを目当てにハックしたコンテンツを生成する方が一定数出てくると思っています

今もスパムユーザーの数が相当多いことを考えると、技術的に良い記事が金銭目的のハック記事などで埋もれてしまう可能性などはどうしても否めず……

あとこれは純粋に運営的な問題なのですが、実は記事に対しての投げ銭を行うためには法律まわりでクリアしなければならないこともあり、本来完全にホワイトに実施できる業者は日本でもそこまで多くないようでその業者にQiitaがなれるのかという問題もあります

そこをしっかりクリアするのか、法的に問題ない別な手段を提供するのか、そういった話もかなり難しいものとなっており……

なんとも要領を得ない回答になってしまい申し訳ありません 🙇

**Comment by blono** [comment]
何らかの方法で収益化できる方法は、ユーザーの目線からいえば、正直かなり欲しいです。
この半年くらいから、Google の AI がサジェストしてくる記事が Zenn で書かれている頻度が徐々に上がってきています。
ここ 2, 3 か月は特に多いですね。

Qiita Pro （独自ドメインで収益化も可能な新しい Qiita）、あってもいいとおもいます！！（課題や労力は非常に多そう😨）

**Comment by tanaka-asami415** [comment]
@munierujp @yimajo @blono 

大変長らくお待たせいたしました🙇
GitHub Sponsors と連携できるようにいたしました！
https://blog.qiita.com/github-sponsors/

法的な確認や社内での議論を重ね、GitHub Sponsors との連携という形でリリースさせていただくことになりました。
みなさまに「感謝や応援の気持ちを伝える新しい方法」として活用いただくことで、
投稿者の方のモチベーションに寄与できればいいなと考えております。ぜひご活用ください✨
